src/main/java/uk/gov/hmcts/ccd/data/definition/DefaultCaseDefinitionRepository.java

```diff
@@ -213,6 +216,24 @@ public JurisdictionDefinition getJurisdictionFromDefinitionStore(String jurisdic
         return jurisdictionDefinitions.get(0);
     }
 
+    @Override
+    public Optional<List<String>> getAllCaseTypesByJurisdictions(List<String> jurisdictionIds) {
+
+        final List<JurisdictionDefinition> jurisdictionDefinitions = getJurisdictionsFromDefinitionStore(jurisdictionIds);
```

Comment:
can you change you IDE to remove automatic adding `final` please and remove it here and the other places

Reply:
DONE